automaxlair/MaxLairInstance.py

```diff
@@ -410,9 +410,12 @@ def calculate_ore_cost(self, num_resets: int) -> int:
         """Calculate the prospective Dynite Ore cost of resetting the game."""
         return 0 if num_resets < 3 else min(10, num_resets)
 
-    def check_sufficient_ore(self, num_resets: int) -> bool:
+    def check_sufficient_ore(self, current_resets_count: int, aditionnal_reset_count: int) -> bool:
         """Calculate whether sufficient Dynite Ore remains to quit the run without saving."""
-        return self.dynite_ore >= self.calculate_ore_cost(num_resets)
+        ore_after_resets = self.dynite_ore
+        for i in range(aditionnal_reset_count):
+            ore_after_resets -= self.calculate_ore_cost(current_resets_count + 1 + i)
+        return 

```

Comment:
I think you probably need to return True or False. One way you could do it is:
`total_ore_cost = 0`
`for i in range(additional_reset_count):`
    `total_ore_cost += self.calculate_ore_cost(self.consecutive_resets + i + 1)`
`return self.dynite_ore > total_ore_cost`

Reply:
forgot to save the file before commiting, the right change was in the commits after :D